package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "bootstrap-social",
+  "version": "4.9.0",
```

Comment:
Sadly, I don't think that can be done: https://docs.npmjs.com/files/package.json#version

Are you only git tagging when releasing at the moment? If so, there is the `npm version` command that updates the `package.json` with the new version and git tags automatically at once: https://docs.npmjs.com/cli/version